marm-mcp-server/tests/test_memory_database.py

```
@@ -280,3 +280,132 @@ async def test_mismatched_embedding_dimension_skipped_with_signal_without_breaki
+                    str(uuid_module.uuid4()),
+                    "no-trunc-test",
+                    f"entry {i}",
+                    vec.tobytes(),
+                    f"hash-notrunc-{i}",
+                ),
+            )
+
+    query_vec = np.ones(dim, dtype=np.float32)
+    query_vec /= np.linalg.norm(query_vec)
+
+    results, meta = await mem.recall_similar(
```

Ruff (0.15.15)

[warning] Unpacked variable `results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)